corehq/apps/hqwebapp/models.py

```diff
@@ -833,7 +833,7 @@ def is_viewable(self):
 
 class MessagingTab(UITab):
     title = ugettext_noop("Messaging")
-    view = "corehq.apps.sms.views.compose_message"
+    view = "corehq.apps.sms.views.default"
```

Comment:
The old referenced view doesn't exist anymore and was replaced by a class-based view as part of the b3 upgrade.  The default view just redirects to the new class-based view.